catalyst/dl/callbacks/base.py

```diff
@@ -135,6 +135,65 @@ def on_stage_end(self, state):
         print(top_best_metrics_str)
 
 
+class IterationCheckpointCallback(Callback):
```

Comment:
For documentation reasons, we need this new Callback to our docs.rst, like https://github.com/catalyst-team/catalyst/blob/master/docs/api/dl.rst